src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultMulti/styles.ts

```diff
@@ -0,0 +1,89 @@
+import styled from '@emotion/styled';
+import {flexibleRowDiv} from '../../shared';
+import * as C from '../colors';
+import {SM} from '../fonts';
+import Icon from '../../../ignitus-Utilities/Components/icon';
+
+
+export const InputContainer = styled.div`
+  width: 528px;
+  height: 138px;
+  left: 456px;
+  top: 116px;
+  background: #FFFFFF;
+  box-shadow: 0px 4px 10px rgba(0, 0, 0, 0.18);
+  border-radius: 16px;
+`;
+
+export const container1 = styled.div`
+  flex: 2;
+`;
+
+export const container2 = styled.div`
+  flex: 1;
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+`;
+
+export const Input = styled.textarea`
+  border: none;
+  font-size: ${SM};
+  outline: none;
+  width: 90%;
```

Comment:
use flex:1 instead by defining its parent as flex, I have updated this as well in the PR :)